src/schema.rs

```diff
@@ -187,20 +187,26 @@ impl TryCmd {
                         // Assuming a trycmd block
                         break;
                     } else {
-                        let mut info = raw.split(',');
-                        let lang = info.next().unwrap();
-                        match lang {
-                            "trycmd" | "console" => {
-                                if info.any(|i| i == "ignore") {
-                                    snapbox::debug!("ignore from infostring: {:?}", info);
-                                } else {
-                                    break;
-                                }
-                            }
-                            _ => {
-                                snapbox::debug!("ignore from lang: {:?}", lang);
+                        let info = raw.split(',').map(|lang| lang.trim());
+                        let mut ignore = false;
+                        let mut accept = false;
+
+                        for lang in info {
+                            if lang == "ignore" {
+                                ignore = true;
+                                break;
+                            } else if lang == "trycmd" || lang == "console" {
+                                accept = true;
                             }
                         }
```

Comment:
This was implemented based on the commonmark spec which said

> The first word of the [info string](https://spec.commonmark.org/0.30/#info-string) is typically used to specify the language of the code sample, and rendered in the class attribute of the code tag. However, this spec does not mandate any particular treatment of the [info string](https://spec.commonmark.org/0.30/#info-string).

In what cases is the lang in other positions and what tools support that?

Reply:
Exactly. The language is not `trycmd`, it's bash. So you want

    ```bash,trycmd

Reply:
`trycmd` was meant to be its own language as this is a custom syntax rather than allowing a custom language to opt-in to trycmd.  If you want to do that, let's move that discussion to an issue.

If this does end up moving forward, the code needs to match commonmark
- The elements of `info` are not `lang` (`lang` used in the map and the for-loop)
- We should be checking `console` against the `lang` and not as any item

Reply:
Ohhh, the code makes more sense now. I thought the idea was that you would eventually be able to use `rust,trycmd` and have it compile and run your rust code. Or interpret python, etc. I think that's way more powerful but yeah needs proper design.